src/Microsoft.DotNet.Archive/LZMA/ICoder.cs

```diff
@@ -0,0 +1,157 @@
+// ICoder.h
```

Comment:
I wasn't suggesting our copyright header should be slapped on.  I am just ensuring that we are doing "the right thing" here.  Copying someone else's code into an OSS project is never a trivial task IMO. In these situations, checking and re-checking is always good.

And maybe looping in someone who doesn't start their response with "I'm no lawyer, but" would be good. :wink: